tests/codegen/default-requires-uwtable.rs

```diff
@@ -1,9 +1,9 @@
-//@ revisions: WINDOWS ANDROID
+//@ revisions: WINDOWS_ ANDROID_
```

Comment:
Why do these ones need to be renamed? They are not builtin.

Reply:
bc compiletest normalize them to lowercase cfg, cf. https://github.com/rust-lang/rust/blob/2c243d957008f5909f7a4af19e486ea8a3814be7/src/tools/compiletest/src/runtest.rs#L682-L684